### ISSUE TITLE
Traffic light detection :how to label the traffic lights in the image?

### ISSUE THREAD

**Issue by JackRen88** [carla-simulator/carla#1045]
Hello,everyone!
In the traffic light detection, I want to label the traffic lights in the image taken from Carla in order to train my detection model,I don't know if there is a way to get the annotation image in Carla.if so,Is it possible to provide pixel coordinates  at four points of bounding box labeled in the image and traffic lights status  which is green or red,orange?

I hope to get your help.

**Comment by chowkamlee81** [comment]
@JackRen88 , did you got to any clue on the smae since iam also struggling

**Comment by 1453042287** [comment]
@JackRen88 @chowkamlee81 why not just use the image of semantic segmentation ?

**Comment by micmarty** [comment]
Anything changed? How to easily automate traffic light dataset gathering process for object detection purposes (ground truth bounding box on camera rgb image)?

My plan was to use Tensorflow Object Detection API :)
